### ISSUE TITLE
Accordian tabs use a hard-wired h3 heading

### ISSUE THREAD

**Issue by Charlie243** [primefaces/primefaces#1630]
The accordion puts the title of each tab into a hard-wired h3 HTML heading -- which can cause structurally incorrect HTML.

I ran into this issue when my web app was evaluated for Section 508 accessibility compliance by a U.S. federal agency. The problem is structural, not stylistic. h1, h2, h3, etc. headings must be in the proper order. You cannot, for example, have an h3 heading follow an h1 without an intervening h2.

It really is a problem that the accordion tabs use a hard-wired h3 heading since it makes all sorts of assumptions about the structure of your HTML. In my case I was able to work around this issue by adding an h2 heading before each accordion panel so all the headings were in the correct order. But this isn't always feasible.

My suggestion would be to not apply any level of heading to the accordion tabs.  (Although another option might be to make the heading level an attribute.)  But, in any case, hardwiring the heading level is definitely not a good policy.

The original discussion in the PrimeFaces forum can be found at: 

http://forum.primefaces.org/viewtopic.php?f=3&t=46412&p=144658#p144658

Thanks,
Charlie

**Comment by kukel** [comment]
I'm in favour of a level. Heck, I'm even more in favour of adding 'nesting' to the accordion by design ;-)

**Comment by mertsincan** [comment]
h3 tag of accordion header was replaced with div tag.
